src/journal/journalctl.c

```diff
@@ -245,7 +245,9 @@ static int help(void) {
                "     --namespace=NAMESPACE   Show journal data from specified journal namespace\n"
                "\n%3$sFiltering Options:%4$s\n"
                "  -S --since=DATE            Show entries not older than the specified date\n"
+               "                               Follow the systemd.time specification\n"
                "  -U --until=DATE            Show entries not newer than the specified date\n"
+               "                               Follow the systemd.time specification\n"
```

Comment:
also, we do not really add those extra 2 indentation chars elsewhere. Please don't do so here either.

Reply:
Indeed. Then, if you'd like, could you also fix the indentation of the lines?